CHANGELOG.rst

```diff
@@ -7,6 +7,13 @@
 Changelog
 =========
 
+3.1.11 - 2023-XX-XX
```

Comment:
```suggestion
Unreleased
```

Reply:
I don't really see the benefits of adding the year, or the minor version. Simply Unreleased here will do.